tests/testit/test-tidy.R

```diff
@@ -4,17 +4,15 @@ tidy.res = function(x, ...) {
   tidy_source(text = x, ..., output = FALSE)$text.tidy
 }
 
-assert(
-  'tidy_source() tries to keep R comments',
-  identical(tidy.res('1+1#asdf'), '1 + 1  #asdf'),
-  identical(tidy.res('paste(1 #asdf\n,2)'), 'paste(1  #asdf\n, 2)'),
-  identical(tidy.res(c('# asdf', '1+1')), c('# asdf', '1 + 1'))
-)
-
-assert(
-  'tidy_source() preserves backslashes in comments',
-  identical(tidy.res('# \\a \\b \\c'), '# \\a \\b \\c')
-)
+assert('tidy_source() tries to keep R comments', {
+  (identical(tidy.res('1+1#asdf'), '1 + 1  #asdf'))
```

Comment:
Could you also change these `identical(A, B)` calls to `A %==% B`?

Reply:
Sure.

Reply:
The computer I'm using is not set up for building packages. Will figure out why CI checks failed.

Reply:
pretty stupid error. fixed.